requirements.txt

```diff
@@ -5,7 +5,7 @@ ipdb>=0.12
 # testing tools
 pylint>=2.3.1
 coverage>=4.5.4
-pytest>=5.3.5
+pytest>=6.0.0
```

Comment:
[`--no-print-logs` needs to be removed for `pytest` 6.0](https://github.com/aws-cloudformation/cloudformation-cli/pull/491)